src/pages/components/Menu.mdx

```diff
@@ -46,49 +47,16 @@ You can modify `offset` and [`placement`](https://popper.js.org/docs/v2/construc
           <span>Open menu</span>
           <Icon as={ChevronDownIcon} size="lg" />
         </Button>
-        <Menu.List offset={0} placement="right-start">
+        <Menu.List sideOffset={12} side="right" align="start">
           {Array(20)
             .fill(null)
             .map((el, i) => (
-              <Item key={i} onClick={() => window.alert(i)}>
+              <Menu.Item key={i} onSelect={() => window.alert(i)}>
                 Item {i}
-              </Item>
+              </Menu.Item>
             ))}
         </Menu.List>
       </Menu>
     </Box>
   </ThemeProvider>
 </Playground>
-
-## Issues
-
-- `ArrowUp` and `ArrowDown` just open the menu, first option is always focused
-- No <kbd>Home</kbd>, <kbd>End</kbd>, <kbd>Page Up</kbd> or <kbd>Page Down</kbd> support
-- No typeahead support
-
-<Playground>
-  <Box p="$8">
-    <Menu>
-      <Button variant="outline">
-        <Icon as={ChevronDownIcon} size="lg" />
-      </Button>
-      <Menu.List>
-        <Dialog>
-          <Item css={{ color: '$danger' }}>
-            <Icon size="md" as={TrashIcon} />
-            <span>Delete</span>
-          </Item>
```

Comment:
First `child` of `Dialog` gets `DIalog.Trigger` props. At the same time, `Menu.Item` has own props, I am afraid that the composition should be turned around, for `Dialog.Trigger` to get all the event listeners assigned first and prevent the behaviour assigned by `Menu.Item`. This unfortunatelly means changing the API of the `Dialog` component to actually expose `Dialog.Trigger` to be rendered, instead of implicit way as of right now